AbstractFactory_AppleSamsung/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace AbstractFactory_AppleSamsung
+{
+    class Program
+    {
+        interface IabstractFactory
+        {
+            void CreateMobile();
+            void CreateLabtop();
+        }
+
+        class ConcrteFactoryApple : IabstractFactory
+        {
+            public void CreateMobile() { Console.WriteLine("Apple phone"); }

```

Comment:
When you create a mobile from Apple Factory, you need to really initiate it and return it to the client.
You made `ProductAppleMobile` class, but you haven't used it.
here, you should return a product means you need to return `new` Object from `ProductAppleMobile`